static/app/components/group/groupSummary.tsx

```diff
@@ -256,6 +256,9 @@ export function GroupSummary({
                 size: 'xs',
                 borderless: true,
                 showChevron: false,
+                style: {
+                  zIndex: 0,
+                },
```

Comment:
i hear there's a preference for styled components over inline styles but I won't be the grammar police

Reply:
yeah but this is a prop, sadly